Source/Core/DolphinQt/DiscordJoinRequestDialog.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+#include <ctime>
+
+class QGridLayout;
+class QPixmap;
+
+class DiscordJoinRequestDialog : public QDialog
+{
+  Q_OBJECT
+public:
+  explicit DiscordJoinRequestDialog(QWidget* parent, const char id[],
+                                    const std::string& discord_tag, const char avatar[]);
+  const std::time_t GetCloseTimestamp();
+
+signals:
+
+private:
+  void CreateMainLayout(const std::string& discord_tag, const QPixmap& avatar);
+  void ConnectWidgets();
+  void Reply(int reply);
+
+  QGridLayout* m_main_layout;
+  QPushButton* m_invite_button;
+  QPushButton* m_decline_button;
+  QPushButton* m_ignore_button;
+
+  const char* const m_user_id;
+  const std::time_t m_close_timestamp;
+  static constexpr std::time_t m_max_lifetime = 30;  // Seconds
```

Comment:
This can just be a constant in the cpp file as: `constexpr std::time_t s_max_lifetime_seconds = 30;`

which gets rid of the need to leave a comment about the units.